sympy/solvers/ode/tests/test_single.py

```diff
@@ -321,6 +321,19 @@ def test_SingleODESolver():
     assert problem.order == 4
 
 
+def test_separable_reduced():
+    df = f(x).diff(x)
+    eq = (x / f(x))*df  + tan(x**2*f(x) / (x**2*f(x) - 1))
+    assert classify_ode(eq) == ('separable_reduced', 'lie_group',
+        'separable_reduced_Integral')
+    _ode_solver_test(_get_examples_ode_sol_separable_reduced())
+
+
+@slow
+def test_slow_examples_separable_reduced():
+    _ode_solver_test(_get_examples_ode_sol_separable_reduced(), run_slow_test=True)
+
```

Comment:
@oscarbenjamin I wanted a suggestion for this like when we make functions for running slow examples then it tests every example for that solver irrespective of slow flag so should I keep it same or change to test only examples which has slow flag true in that solver?

Reply:
It should only the tests the ones with the slow flag.